Core/DISET/private/Transports/SSL/SocketInfoFactory.py

```diff
@@ -22,6 +22,15 @@
 
 class SocketInfoFactory:
 
+  def __init__(self):
+    self.__timeout = 1
+  
+  def setSocketTimeout(self, timeout):
+    self.__timeout = timeout
+    
+  def getSocketTimeout(self):
+    return self.__timeout
+  
```

Comment:
In that case I would like to use private variable. If someone want to change it, it has to use the setter.